<?xml version="1.0" ?>
<launch>
	<!-- <node pkg="nodelet" type="nodelet" name="primesense_nodelet" args="manager"/> -->
	<node pkg="tf" type="static_transform_publisher" name="primesense_broadcaster" args="0.05 0 0.13 0 0 0 fcu camera_link 100"/>

	<node pkg="nodelet" type="nodelet" name="octomap_server_nodelet" args="load octomap_server/OctomapServerNodelet aeplanner_nodelet_manager">
                <param name="resolution" value="0.1" />
		<param name="publish_free_space" value="full" />
		
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="world" />
		
		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model/max_range" value="5" />
	</node>
	<node pkg="nodelet" type="nodelet" name="primesense_to_pointcloud_nodelet" args="load rpl_exploration/PTPNodelet aeplanner_nodelet_manager">
		<param name="scan_topic" type="string" value="depth/points" />
		<param name="cloud_topic" type="string" value="cloud_in" />
		<param name="cloud_skip" value="30" />
	</node>
</launch>

